File: Musical-Broccoli-App/app/src/main/java/com/cetys/dreamteam/musicalbroccoli/commons/dtos/RatingDto.java
package com.cetys.dreamteam.musicalbroccoli.commons.dtos;

/**
 * Created by Octavio on 2016/11/12.
 */

public class RatingDto {
    private int id;
    private int ratingValue;
    private String comment;
    private int userId;
    private int tourId;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getRatingValue() {
        return ratingValue;
    }

    public void setRatingValue(int ratingValue) {
        this.ratingValue = ratingValue;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getTourId() {
        return tourId;
    }

    public void setTourId(int tourId) {
        this.tourId = tourId;
    }
}

File: Service/Musical.Broccoli.API/src/Common/DTOs/RatingDTO.cs
namespace Common.DTOs
{
    public class RatingDTO : BaseDTO
    {
        public override int Id { get; set; }
        public int RatingValue { get; set; }
        public string Comment { get; set; }
        public int UserId { get; set; }
        public int TourId { get; set; }

        public UserDTO User { get; set; }
        public TourDTO Tour { get; set; }
    }
}